packages/typespec-azure-portal-core/.mocharc.yaml

```diff
@@ -0,0 +1,4 @@
+timeout: 5000
```

Comment:
Over the holidays we migrated from mocha to vitest, you'll have to do the same here. 

here is the PR for reference of what to do https://github.com/Azure/typespec-azure/pull/54

It should be:
- remove all the mocha files and reference to mocha
- add the vitest.config.ts file like the other packages 
- update the tsconfig.json to include `vitest.config.ts`
- update thje package.json to use vitest for test instead of mocha

Reply:
converted to vitest :) thank you so much Tim!